Docs 9546 edit pm config topics esb (#99)
mulesoft/docs-partner-manager: modules/ROOT/pages/Certificates.adoc

= Certificate Settings

> All certificates in Anypoint Partner Manager are securely stored in Anypoint Platform’s secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.

Reviewer: ```suggestion
All certificates in Anypoint Partner Manager are securely stored in the Anypoint Platform secrets manager, which uses secure vault technology to store and to control access to private keys, passwords, certificates, and other secrets.
```
...to avoid making the (R) possessive

> You can configure your SSL certificate settings based on your certificate needs.

Reviewer: Per style ref., try to avoid modal verbs like "can," may," and "might" unless you are trying to distinguish between a step and an option.

Reviewer: Does one configure "settings"? Aren't settings themselves configuration?
And "configure your certificate based on your certificate needs" sounds kind of...unnecessary. What are we really trying to advise here?

Author: Fixed - superfluous statement that was consolidated into the next line.

> The information provided for each certificate includes:

Reviewer: Maybe this sentence can be reworded to replace the previous gratuitous statement:
```suggestion
Configure your certificate to include the following information:
```

* *Authority*: The certificate provider
* *Serial Number*: The serial number from the certificate

> * *Status*: The status of the certificate. The color coded icon on the certificate shows the user the status of the certificate:

Reviewer: ```suggestion
* *Status*: The status of the certificate. 
+
The color-coded icon on the certificate shows the user the status of the certificate:
```

** Green: The certificate is active

> ** Red: The certificate is inactive

Reviewer: periods

Author: Done.

> * *Start Date*: The date from when the certificate is valid

Reviewer: ```suggestion
* *Start Date*: The date from which the certificate is valid
```

* *Expires*: The date after which the certificate is no longer valid
* *Endpoints*: The number of endpoints using the certificate

> * *Message Flows*: Number of message flows using the certificate

Reviewer: ```suggestion
* *Message Flows*: The number of message flows using the certificate
```
...for parallelism with previous

> In Partner Manager, you can:

Reviewer: This is a very vague introduction. Can we reword to somehow introduce the precise information following: certificate types and connection settings?

Author: Done - added heading

* Use a CA-signed certificate. +
If an endpoint is currently using a CA-signed certificate, you do not need to upload an additional certificate. +
* Upload your own, self-signed certificate. +

> You can upload a new certificate (in `<certifcate name>.PEM` format), or view all the certificates available for use with this endpoint setup.

Reviewer: ```suggestion
You can upload a new certificate (in `<certifcate name>.PEM` format) or view all the certificates available for use with this endpoint setup.
```

> * Enable an insecure connection for local testing that doesn’t perform certificate validation by selecting *Allow insecure connection*.

Reviewer: It's a bit difficult for me to parse the previous text and the new text, but it looks like this former 4-item list is now a 3-item list? Would you please check?

Author: There was one item that was removed because it was no longer valid and does not appear in the UI.

== Add Certificates

> You can add certificates from the Host or Partner page.

Reviewer: Why are these page types init. capped?
If these are UI labels, then they require * *.

Author: Once again, nameless pages. Changed to lower case.

> . In Partner Manager's left navigation, select either *<host name>* or *Partners*. +

Reviewer: Do we have to specify "Partner Manager's"? 
Is this just the home page, or does it have a label?

Author: Done

> If you select Partners, select the parter to which you want to add the certificate from the list.

Reviewer: ```suggestion
If you select *Partners*, select from the list displayed the partner to which you want to add the certificate.
```

. In the *Certificates* section, click *New*.

> . In the *New certificate for <host name>* or the *New certificate for Partners* window, in the *Certificate* field, click *Choose file* to upload a certificate.

Reviewer: ```suggestion
. In the *Certificate* field of the *New certificate for <host name>* or the *New certificate for Partners* window, click *Choose file* to upload a certificate.
```

. Click *Upload Certificate*. +
The new certificate is added to the host or the trading partner.

== Delete Certificates

> You can delete certificates that are not deployed or associated with endpoints.
> 
> You cannot delete a certificate if it is the only certificate that is uploaded.

Reviewer: ```suggestion
You can delete certificates that are not deployed or associated with endpoints; however, you cannot delete a certificate if it is the only certificate that is uploaded.
```

To delete a certificate:

. In Partner Manager's left navigation, select *<host name>* or *Partners*. +

> If you select Partners, select the parter to which you want to add the certificate from the list.

Reviewer: ditto prev. edits to same text

Author: Done

. In the *Certificates* section, click the certificate to delete.

> . In the dialog window, click *Delete*.

Reviewer: dialog window? Isn't "a dialog" typically called "a dialog box"?

Author: Done
